app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_save :update_likes_counter
+
```

Comment:
- [ ] Nice use of of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save`. Kudos 👍